binderhub/templates/settings.html

```diff
@@ -0,0 +1,47 @@
+{% extends "page.html" %}
+
+{% block head %}
+{{ super() }}
+<script src="{{static_url("settings.js")}}"></script>
+{% endblock head %}
+
+{% block main %}
+<div id="main" class="container">
+  <div class="row">
+    <h4 class="text-center logo-subtext">(beta)</h4>
+    {%if use_as_federation_portal %}
+    <div class="col-lg-10 col-lg-offset-1">
+      <div id="header" class="text-center">
+        <h3>Your Known Binders</h3>
```

Comment:
Your Federation Binder Sites

Here is a list of the binder sites available in your federation.